pkgs/development/python-modules/strawberry-graphql/default.nix

```diff
@@ -0,0 +1,43 @@
+{ lib, buildPythonPackage, fetchFromGitHub, poetry, pythonOlder
+, cached-property, click, graphql-core, pygments, python-dateutil, python-multipart, typing-extensions
+, aiohttp, asgiref, chalice, fastapi, flask, pydantic, sanic, starlette, uvicorn
+}:
+
+buildPythonPackage rec {
+  pname = "strawberry-graphql";
+  version = "0.124.0";
+  format = "pyproject";
+
+  disabled = pythonOlder "3.7";
+
+  src = fetchFromGitHub {
+    owner = "strawberry-graphql";
+    repo = "strawberry";
+    rev = version;
+    sha256 = "sha256-NBQ7/h27WnyAX5KnonBzL4ExipRP5OhT5UyDNjb8wPQ=";
+  };
+
+  patches = [
+    ./fix_cached_property.patch
```

Comment:
Is this from an upstream commit? If so please fetch that. Otherwise we should try to upstream it.

Reply:
It looks like they use another `cached-property` package which in nixos does not have - https://github.com/strawberry-graphql/strawberry/pull/1582
Then need to add this package - https://pypi.org/project/backports.cached-property/ ?

Reply:
Added package backports.cached-property.